plugins/occt/CMakeLists.txt

```diff
@@ -71,6 +71,15 @@ f3d_plugin_declare_reader(
   CUSTOM_CODE "${CMAKE_CURRENT_SOURCE_DIR}/brep.inl"
 )
 
+f3d_plugin_declare_reader(
+  NAME XBF
```

Comment:
does this reader work at all with only TKMesh ?

If not, you may want to hide it behind F3D_PLUGIN_OCCT_COLORING_SUPPORT

Reply:
Safer to hide it indeed

Reply:
please update the cmake warning accordingly if you do so :)

Reply:
what I mean was to update the module checking in `plugins/occt/CMakeLists.txt` and also check for TKBinXCAF